src/resources/demo_db/sqlite/v0.1/create_tables.py

```diff
@@ -0,0 +1,56 @@
+from os.path import join
```

Comment:
This script is here for reference; It will be fetched from S3 during the update process.

Reply:
In future, should each version with new table contains a `create_tables.py` in that version's directory?